Android/app/src/main/java/com/harvard/usermodule/SignupActivity.java

```diff
@@ -84,6 +84,8 @@
   private RegistrationData registrationData;
   private String userAuth;
   private String userID;
+  String passwordPattern =
```

Comment:
This appears to be hard-coded in many places. Is there a reason this isn't centralized somewhere?

Reply:
i've centralized it